feature-libs/order/root/utils/order-card-utils.ts

```diff
@@ -33,10 +33,18 @@ export function deliveryAddressCard(
   }
 
   const numbers = getAddressNumbers(deliveryAddress, textPhone, textMobile);
+  let fullName;
+  if (deliveryAddress.firstName && deliveryAddress.lastName) {
+    fullName = deliveryAddress.firstName + ' ' + deliveryAddress.lastName;
+  } else if (deliveryAddress.firstName) {
+    fullName = deliveryAddress.firstName;
+  } else if (deliveryAddress.lastName) {
+    fullName = deliveryAddress.lastName;
+  }
 
   return {
     title: textTitle,
-    textBold: deliveryAddress.firstName + ' ' + deliveryAddress.lastName,
+    textBold: fullName,
```

Comment:
[MEDIUM]
1. What if both first and last name are `undefined`? Then `fullName` is `undefined` and it will be presented in the UI I guess? What should we present instead?

2. Let's add unit tests describing the expected behaviors